docs/guides/agent-workflow.md

```
@@ -0,0 +1,110 @@
+    "Visual verify with Playwright MCP" -> "Visual OK?";
+    "Visual OK?" -> "Fix errors" [label="no"];
+    "Visual OK?" -> "E2E needed?" [label="yes"];
+    "E2E needed?" -> "Write spec + run bun run verify:e2e" [label="yes"];
+    "E2E needed?" -> "Commit + show results" [label="no"];
+    "Write spec + run bun run verify:e2e" -> "Commit + show results";
+}
+```
+
+## Verify (mandatory, enforced)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)